README.md

```diff
@@ -23,7 +23,11 @@ refreshing files and (Geo)DataFrames or updating layer styles and element proper
 pip install felt-python
 ```
 
-## Usage
+## Documentation
+
+See the [docs](/docs) directory for Juypter notebooks with complete examples of using the API.
```

Comment:
Love seeing more example notebooks but not sure these qualify as "docs". Maybe we can call them `examples` or `notebooks` or similar?